packages/libircclient/patch-fix-warnings

```diff
@@ -0,0 +1,99 @@
+--- src/colors.c
```

Comment:
Yeah, I preferred using the safe functions instead of the unsafe one, also in the hopes that upstream will apply this.